.github/workflows/tests.yml

```diff
@@ -51,7 +54,8 @@ jobs:
     strategy:
       matrix:
         features:
-          - "--features iouring"
+          - "--features commonware-runtime/iouring-storage"
+          - "--features commonware-runtime/iouring-network"
```

Comment:
Can you also remove here and reference #954

Reply:
I think leaving on `lint` should be fine (and is best) 👍

Reply:
Done. Figured we might want to leave this one since it hadn't flaked.